crates/bevy_transform/src/transform_propagate_system.rs

```diff
@@ -96,33 +99,31 @@ mod test {
         schedule.add_stage("update", update_stage);
 
         // Root entity
-        world.spawn().insert_bundle((
-            Transform::from_xyz(1.0, 0.0, 0.0),
-            GlobalTransform::identity(),
-        ));
+        world
+            .spawn()
+            .insert_bundle(TransformBundle::from_transform(Transform::from_xyz(
```

Comment:
The below series repeating again and again `TransformBundle::from_transform(Transform::from_xyz(...))` suggests maybe it's worth adding some utility `TransformBundle::from_xyz()`?

Reply:
Rather than mirroring the Transform api, maybe we can improve the ergonomics by favoring `TransformBundle::from(Transform::from_xyz())` in cases where const isn't necessary?